idl/group0_tables_lang.idl.hh

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace raft {
+namespace group0_tables {
+
+struct select_query {
+    bytes key;
+};
+
+struct update_query {
+    bytes key;
+    bytes new_value;
```

Comment:
We use managed_bytes for values to avoid large allocations that are difficult for the allocator.

Reply:
I've noticed that the serializer hasn't instantiation for managed_bytes. Should I write it?

Reply:
`atomic_cell`s, which `mutation`s are built of, use `managed_bytes` everywhere (see atomic_cell.hh). But if `managed_bytes` doesn't have a serializer, how are they sent over the network?
So I looked into idl/mutation.idl.hh and saw that they use `bytes`. E.g.:
```
class live_cell stub [[writable]] {
    api::timestamp_type created_at;
    bytes value;
};
```

Reply:
Ah, but it's a `stub`, and in idl-compiler.py you can find the following comment:
```
    The following specifiers are allowed in a class declaration:
     - `final` -- if a class is marked with this keyword it will not contain a
       size argument. Final classes cannot be extended by a future version, so
       it should be used with care.
     - `stub` -- no code will be generated for the class, it's only there for
       documentation.
```

so apparently `live_cell` must have a "manual" serializer definition somewhere.

Reply:
I think you should look for an example of a type that uses `managed_bytes` and is sent over the network - like `mutation`. Then find out how this type is being (de)serialized, and use that approach for your struct.